tests/endtoend/opart/comparison/forward_time_opa_and_art.py

```diff
@@ -0,0 +1,71 @@
+from exojax.spec.opacalc import OpaPremodit
+from exojax.spec.atmrt import ArtEmisPure
+from exojax.utils.grids import wavenumber_grid
+from exojax.spec.api import MdbExomol
+from exojax.utils.astrofunc import gravity_jupiter
+import jax.profiler
+
+from jax import config
+
+config.update("jax_enable_x64", True)
+#from jax_smi import initialise_tracking
+#initialise_tracking()
+
+Nnus = 100000
+nu_grid, wav, resolution = wavenumber_grid(
+    1900.0, 2300.0, Nnus, unit="cm-1", xsmode="premodit"
+#    2050.0, 2150.0, Nnus, unit="cm-1", xsmode="premodit"
```

Comment:
```suggestion
#    1900.0, 2300.0, Nnus, unit="cm-1", xsmode="premodit"
    2050.0, 2150.0, Nnus, unit="cm-1", xsmode="premodit"
```
I think it would be better to use a high-resolution grid for the reason of #545